projects/dotcms/src/lib/models/DotCMSContentType.model.ts

```diff
@@ -38,3 +38,7 @@ export interface DotCMSContentType {
     variable?: string;
     workflow?: string;
 }
+
+export interface DotCMSContent {
+    stName: string;
```

Comment:
I would change this to contentTypeName, stName is an ugly legacy name

Reply:
any chance on changing this on the BE? @freddyucv @fmontes

Reply:
Not in this issue, we have to modify the endpoint, the documentation etc... way out of scope.